scio-core/src/main/scala/com/spotify/scio/values/PairSCollectionFunctions.scala

```diff
@@ -624,6 +626,30 @@ class PairSCollectionFunctions[K, V](val self: SCollection[(K, V)])
     new MapSideInput[K, V](o)
   }
 
+  def asSparkeySideInput: SideInput[Map[String, String]] = {
+    val f = self.groupBy(_ => ()).values
+      .map(_.map(kv => (kv._1.toString, kv._2.toString))).map { iter =>
```

Comment:
Also the 2 `.toString` are scary. I'd rather add view bounds (`T <:< String`) on `K` and `V`